earth_enterprise/rpms/build.gradle

```diff
@@ -520,6 +520,20 @@ task openGeeServerRpm (type: GeeRpm, dependsOn: openGeeSharedFiles) {
     directory('/var/opt/google/pgsql', 0700)
     directory('/var/opt/google/pgsql/logs')
 
+    filesMatching(
+        ['/etc/init.d/geserver']
```

Comment:
Can't we fix these permissions in the repository, instead of adding packaging code?

Reply:
I actually fixed the perms in the repo, but wasn't sure why they were getting the wrong perms/needed to have it being set here in the first place.

Reply:
I was not sure why we were setting these perms in the gradle build in the first place, so I wanted to make sure they were at least correct there, and I did already change the perms in the repo.

Reply:
OK.  I'm not sure what the scope of this pull request is.  If verifying that this code is no longer needed, and removing it is not in scope, it could be done on another ticket.